server/startup/migrations/v156.js

```diff
@@ -0,0 +1,99 @@
+import { Mongo } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations/server';
+import {
+	Messages,
+	Rooms,
+	Subscriptions,
+	Users,
+	Uploads,
+	Settings,
+} from '../../../app/models/server';
+
+const validSettings = [
+	'FEDERATION_Enabled',
+	'FEDERATION_Domain',
+	'FEDERATION_Public_Key',
+	'FEDERATION_Discovery_Method',
+	'FEDERATION_Test_Setup',
+];
+
+const updateLocalUsers = () => {
+	console.log('Migration: update local users');
+	const options = {
+		fields: { _id: 1 },
+		limit: 500,
+	};
+	const users = Users.find({ federation: { $exists: true } }, options).fetch();
+
+	const ids = users.map((u) => u._id);
+	if (ids.length === 0) {
+		return;
+	}
+
+	Users.update({ _id: { $in: ids } }, { $unset: { federation: 1 } }, { multi: true });
```

Comment:
Can't it be just the update?

Reply:
it was breaking the server.. changed to use `rawCollection` and it worked fine.